exercises/custom-set/example.ml

```diff
@@ -1,3 +1,5 @@
+open Core.Std
+
```

Comment:
With the code as it stands, yes. I could probably remove it, but other exercises use core (and the Makefile uses corebuild). It seems best to keep consistent.